lib/ReactCrop.js

```diff
@@ -749,6 +799,34 @@ class ReactCrop extends PureComponent {
       'ReactCrop--crop-invisible': crop && cropIsActive && (!crop.width || !crop.height),
     });
 
+    // In order to call onImageLoad once the renderComponent has been loaded, we use a dummy image with width
+    // and height set to 0. In order to prevent browser warnings, we pass in a base 64 src, which is 1 x 1
+    // transparent gif. Details of this implementation can be found here: https://stackoverflow.com/a/27754238
+    const component = renderComponent 
+    ? (
+      <div
+        ref={(n) => { this.imageRef = n; }}
+        className="ReactCrop__image"
+        style={imageStyle}
+        onError={onImageError}
+      >
+        {renderComponent}
+        <img src='data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==' width='0' height='0' onLoad={e => { this.onImageLoad(e.target)}}/>
```

Comment:
What's the purpose of having this image here?

Reply:
I tried to explain it in the comment above, but essentially it's there to ensure that `onComponentRendered` will be called when we use a custom component. The `onLoad` property only works on img and body tags, so we can't add it to the custom component. Therefore we use a dummy image to force the function to be called when the component is rendered.